app/helpers/ftp_helper.py

```diff
@@ -0,0 +1,45 @@
+from app import settings
```

Comment:
If we're refactoring maybe this should be a class? There's other projects using these same methods where use of a ftp global seems to cause edge cases.